requirements/app/ui.txt

```diff
@@ -1,2 +1,2 @@
-streamlit>=1.3.1, <=1.11.1
+streamlit>=1.0.0
```

Comment:
I would advise against removing the version limit. Please consider bumping the version instead:
```suggestion
streamlit>=1.0.0, <=1.15.2
```